### ISSUE TITLE
Inline equations rendered as text instead of images

### ISSUE THREAD

**Issue by nalimilan** [JuliaDocs/Documenter.jl#479]
Both [in the Documenter manual](https://juliadocs.github.io/Documenter.jl/latest/man/latex.html) and when building the Julia manual locally, inlined equations are rendered as text instead of images. See this example in the previous link:
```
Here's some inline maths: ``\sqrt[n]{1 + x + x^2 + \ldots}``.
which will be displayed as
Here's some inline maths: $\sqrt[n]{1 + x + x^2 + \ldots}$.
```

This seems to be a recent regression.

**Comment by KristofferC** [comment]
For me it depends on the browser used. On my phone it renders fine for example.

**Comment by nalimilan** [comment]
Interesting. Which kind of phone? The equation is rendered as text on Firefox, GNOME Web (WebKit) and Safari (iPhone) here.

**Comment by mortenpi** [comment]
I have a hunch that this is related to the [CDN shutting down](https://www.mathjax.org/cdn-shutting-down/) (#473) and the redirect they have set up. A build with a direct cdnjs URL [seems to work fine](http://mortenpi.eu/Documenter.jl/bundle-highlightjs-deploy/man/latex.html).

So this should also be fixed by #468, once that gets merged and tagged.

**Comment by mortenpi** [comment]
Fixed by #468. There will a patch release shortly.

**Comment by nalimilan** [comment]
Great, thanks!
